shell/components/Tabbed/index.vue

```diff
@@ -32,7 +32,7 @@ export default {
 
     useHash: {
       type:    Boolean,
-      default: true,
+      default: false,
```

Comment:
You have to update all the use cases of Tabbed component if you want to revert this key. 
The alternative is that you set it true for the case you want to apply it.